lib/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ if (EXISTS "RNApuzzler/RNApuzzler")
 	add_subdirectory(RNApuzzler)
 endif()
 
+
```

Comment:
Nit: unnecessary whitespace change

Reply:
the new version that checks if exists is implemented now so this is no longer relevant.

Reply:
took out the extra line